.gitignore

```diff
@@ -18,7 +18,6 @@
 .env.test.local
 .env.production.local
 .watchmanconfig
-package-lock.json
```

Comment:
package-log.json should be commited.
https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json